cmd/kusk/cmd/install.go

```diff
@@ -354,6 +355,7 @@ func installApi(helmPath, releaseName, releaseNamespace, envoyFleetName string)
 		"--set", fmt.Sprintf("fullnameOverride=%s", releaseName),
 		"--set", fmt.Sprintf("envoyfleet.name=%s", envoyFleetName),
 		"--set", fmt.Sprintf("envoyfleet.namespace=%s", releaseNamespace),
+		"--set", fmt.Sprintf("analytics.enabled=%s", analyticsEnabled),
```

Comment:
The GH diff doesn't make this clear but this is me adding the --set flag to the `installKuskGatewayApi` function